python/tvm/relax/backend/contrib/cutlass.py

```diff
@@ -116,7 +173,6 @@ def partition_for_cutlass(mod):
         compiled by the CUTLASS backend.
     """
 
-    cutlass_patterns = get_patterns_with_prefix("cutlass")
-    return transform.FuseOpsByPattern(cutlass_patterns, bind_constants=True, annotate_codegen=True)(
-        mod
-    )
+    cutlass_pattern_entries = get_patterns_with_prefix("cutlass")
+    patterns = [(e.name, e.pattern, e.check) for e in cutlass_pattern_entries]
+    return transform.FuseOpsByPattern(patterns, bind_constants=True, annotate_codegen=True)(mod)
```

Comment:
```suggestion
    return transform.FuseOpsByPattern(patterns, bind_constants=False, annotate_codegen=True)(mod)
```
I made a typo in the previous PR, it should be False